### ISSUE TITLE
The service "sylius.factory.taxon" has a dependency on a non-existent service "sylius.translation.locale_provider".

### ISSUE THREAD

**Issue by itinance** [Sylius/Sylius#4633]
When registering TaxonomyBundle, which is required by WebBundle, the following error occurs:

> [Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]                                           
>   The service "sylius.factory.taxon" has a dependency on a non-existent service "sylius.translation.locale_provider".
> 

In current master branch the TranslationBundle was removed. In 0.17.0 release it is still present and everything works fine. I couldnt find any service related to sylius.translation.local_provider in the current master branch. 

**Update**

In 0.17, the service was "Sylius\Bundle\CoreBundle\Provider\TranslationLocaleProvider".
This class at dev-master is represented by a service-id called "sylius.translation.locale_provider.contextual".

**Update End**

**Update 2**
I've found out that sylius.translation.locale_provider is now a alias for sylius.translation.locale_provider.contextual.
Anyway, there is still a service-id called "sylius.factory.taxon", which requires the service "sylius.translation.locale_provider" at a time when this alias isn't known.
**

My AppKernel looks like this:

```
            new Sylius\Bundle\ResourceBundle\SyliusResourceBundle(),

            new Sylius\Bundle\ReportBundle\SyliusReportBundle(),

            new Sylius\Bundle\MoneyBundle\SyliusMoneyBundle(),
            new Sylius\Bundle\CurrencyBundle\SyliusCurrencyBundle(),
            new Sylius\Bundle\LocaleBundle\SyliusLocaleBundle(),
            new Sylius\Bundle\AttributeBundle\SyliusAttributeBundle(),

            new Sylius\Bundle\TaxonomyBundle\SyliusTaxonomyBundle(),
            new Sylius\Bundle\WebBundle\SyliusWebBundle(),

            new Sylius\Bundle\RbacBundle\SyliusRbacBundle(),
            new Sylius\Bundle\UserBundle\SyliusUserBundle(),
            new Sylius\Bundle\UiBundle\SyliusUiBundle(),

            new Sylius\Bundle\MailerBundle\SyliusMailerBundle(),

            new Sylius\Bundle\AdminBundle\SyliusAdminBundle(),
            new Sylius\Bundle\GridBundle\SyliusGridBundle(), 

```

I added dev-master branch via composer:

```
    "require" : {
        ...
        "sylius/sylius": "dev-master",
        ...
    }

```

_(My reason of using dev-master is the need for GridBundle, which isn't part of current release 0.17.0 ;))_

**Comment by itinance** [comment]
Update, FYI:

In 0.17, the service was "Sylius\Bundle\CoreBundle\Provider\TranslationLocaleProvider".
This class at dev-master is represented by a service-id called "sylius.translation.locale_provider.contextual".

**Comment by itinance** [comment]
**Update 2**
I've found out that sylius.translation.locale_provider is now a alias for sylius.translation.locale_provider.contextual.
Anyway, there is still a service-id called "sylius.factory.taxon", which requires the service "sylius.translation.locale_provider" at a time when this alias isn't known.
**

**Comment by liverbool** [comment]
Are you mixed sylius version? Or why you not use sylius-core kernel if you install whole sylius packages?

There are too many services you need to register and config when you are using many packages. 

Looking to:
https://github.com/Sylius/Sylius/tree/master/app/config/config.yml

https://github.com/Sylius/Sylius/tree/master/src/Sylius/Bundle/CoreBundle/Resources/config

https://github.com/Sylius/Sylius/tree/master/src/Sylius/Bundle/CoreBundle/Resources/config/app

**Comment by adamelso** [comment]
I have a PR to move the TranslationLocaleProvider out of the CoreBundle, to... er, somewhere yet to be determined - looks like it should be at a lower-level than the Core. https://github.com/Sylius/Sylius/pull/3940

**Comment by JulianBustamante** [comment]
I solved this issue adding the following configuration in the config.yml file:

```yml
sylius_resource:`
    translation:
        enabled: true
        default_locale: "%locale%"
```

**Comment by TheMadeleine** [comment]
Fixed by #5974, and the deletion of WebBundle 🎉 

@pjedrzejewski, @michalmarcinkowski
